java-source/src/test/java/com/example/contract/IOUFlowTests.java

```diff
@@ -52,8 +52,7 @@ public void flowReturnsASignedTransaction() throws InterruptedException, Executi
         ListenableFuture<SignedTransaction> future = a.getServices().startFlow(flow).getResultFuture();
         net.runNetwork(-1);
 
-        Object signedTx = future.get();
-        assert(signedTx instanceof SignedTransaction);
+        SignedTransaction signedTx = future.get();
```

Comment:
Screw it, I'll drop the test. The compiler will force people to return a SignedTransaction anyway.